markdown/developers/training_guide.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Training material guide
+subtitle: A guide to writing instructions for training material.
+---
+
+# Contributing a training guide to nf-core
+
+Thank you for helping us write a guide to using an aspect of nf-core.
+
+## Links
+
+- [Questions](#questions)
+- [Objectives](#objectives)
+- [Glossary](#glossary)
+- [Before I begin](#before-i-begin)
+- [How to](#how-to)
+  - [Bytesize](#bytesize-talk)
+  - [Walkthrough](#walkthrough)
+- [Key points](#key-points)
+- [References](#references)
+
+## Questions
+
+- How can I contribute a training guide to nf-core?
+- Where do I put the guide?
+- How do I structure content?
+
+## Objectives
+
+- Learn how to structure an nf-core guide.
+- Know where to submit the pull request.
+
+## Glossary
+
+<details>
+    <summary>Click here to expand the glossary of potentially unfamiliar terms</summary>
+    - **Learning Objective:** A predictive statement describing the specific outcomes that a training session is intended to achieve.
+    - **Key point:** An important take home message.
+    - **Cognitive load:** This relates to the amount of information that working memory can hold at one time.
+    - **Fork a repository:** This makes a copy of the repository to your personal GitHub workspace.
+    - **Make a branch:** A branch is another version of the files in your repository, including a history of how files were changed from the start of the branch to the current state of files.
+</details>
+
+## Before I begin
+
+A checklist of what you need to begin.
+
+- Ask on #training about the guide you'd like to contribute.
+- Make an issue on [nf-core website issues](https://github.com/nf-core/nf-co.re/issues) describing what you want to do, and assign yourself.
+- GitHub account.
+
+## How to
+
+### Bytesize talk
+
+There is no bytesize talk at present, but if you'd like present one, please make a suggestion on the [Slack #bytesize channel](https://nfcore.slack.com/channels/bytesize).
+
+### Walkthrough
+
+#### Use this as a template.
+
+Use this file `https://github.com/nf-core/nf-co.re/markdown/developers/training_guide.md` as a template to guide you.
+
+1. If you've not done so, make a fork of the [nf-core website repository](https://github.com/nf-core/nf-co.re/).
+2. Make a new branch from the `master` branch in your fork named after the guide you indend to submit e.g., `gitpod_training_guide`.
+3. Make a copy of the `training_guide.md` with an appropriate name to use as your skeleton.
+
+#### Add questions.
+
+Include between 1-3 questions your guide is intended to answer.
+
+#### Add objectives.
+
+Objectives are intended to describe the outcomes someone reading your guide should be able to do once they've read through it.
+The objectives also function as a checklist of things that should be part of the guide.
+
+#### Add a glossary.
+
+The glossary should define potentially unfamilar terms to a novice trying to use your guide for the first time.
+
+#### Prepare your reader.
```

Comment:
Would these headings be better as bullet points? They're basically all boil down to "fill in the sections at the top of this file".

Reply:
I think they would be better as sections because then it's easier to include visual aids and other stuff. This was a very quick mock up, and would likely benefit from more detail.